velox/common/encode/tests/Base64Test.cpp

```diff
@@ -86,4 +89,14 @@ TEST_F(Base64Test, calculateDecodedSizeProperSize) {
   EXPECT_EQ(14, encoded_size);
 }
 
+TEST_F(Base64Test, checksPadding) {
```

Comment:
nit: consider renaming these test to match the methods they are testing: isPadded and numPadding.